### PR TITLE
Add Japanese language file

### PR DESCRIPTION
**Pull request by shacho36p**
Hello.

I made Japanese translation file for XPERI+.
Therefore, I will send it to you .

I'm sorry I'm unused to GitHub.

### CODE REVIEW

File: app/src/main/res/values-ja/plurals.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <plurals name="approved_link_list_title">
```
Comment (ivaniskandar) [inline app/src/main/res/values-ja/plurals.xml:3]: This one

**Review by ivaniskandar — CHANGES_REQUESTED** [review]
Hi! Thanks for the translation. I'll handle the rest but FYI you've missed these file to translate. You can add it by yourself or put it in the comment here.

**Comment by ivaniskandar** [comment]
New string to translate:
`<string name="digital_assistant_action_label">Digital Assistant</string>`

**Review by shacho36p — COMMENTED** [review]
I have translated the files that were not translated and updated the strings.xml.
Thank you for your cooperation.

**Review by ivaniskandar — APPROVED** [review]

**Comment by ivaniskandar** [comment]
Thanks for your help!

**Pull request merged**